src/godot_manager.rs

```diff
@@ -505,10 +505,14 @@ impl<'a> GodotManager<'a> {
                 }
                 if console {
                     // On macOS, running attached is the default behavior
-                    std::process::Command::new(inner_path).spawn()?;
+                    std::process::Command::new(inner_path)
+                        .args(godot_args)
+                        .spawn()?;
                 } else {
```

Comment:
these branches should probably be different, but it's not impacting me at the moment

Reply:
It should be fine for now as is, but yes I haven't looked more closely at how this would be different on the macOS platform yet.